pages/chaoss/visualizations/contrib_drive_repeat.py

```diff
@@ -31,26 +34,26 @@
                             have the critera of how many contributions it takes for a member to be a repeat contributor."
                         ),
                     ],
-                    id="chaoss-popover-1",
-                    target="chaoss-popover-target-1",  # needs to be the same as dbc.Button id
+                    id=f"{PAGE}-popover-{VIZ_ID}",
+                    target=f"{PAGE}-popover-target-{VIZ_ID}",
                     placement="top",
                     is_open=False,
                 ),
                 dcc.Loading(
-                    dcc.Graph(id="cont-drive-repeat"),
+                    dcc.Graph(id=f"{PAGE}-{VIZ_ID}"),
                 ),
                 dbc.Form(
                     [
                         dbc.Row(
                             [
                                 dbc.Label(
                                     "Contributions Required:",
-                                    html_for="num_contributions",
+                                    html_for=f"{PAGE}-contributions_required-{VIZ_ID}",
```

Comment:
This is everywhere throughout the template but I'm making a comment here- let's standardize these to snake-case (with a hyphen in between these variables rather than an underscore and hyphen thing).

Reply:
I personally made this choice to differentiate between the build in page and viz identifiers and the actual title of the callback. I would rather it be this way, but open for discussion if this is something you are passionate about as well

Reply:
It's just uncommon naming hygiene to mix hyphens and underscores. the 'correct' typical multi-word naming regimes are camelCase, kebab-case, and snake_case; i've never seen them mixed. could differentiate w/ capitalization, e.g. 
"CONTRIBUTIONS-REQUIRED-{PAGE}-{VIZ_ID}"

Reply:
Makes sense, I went with "contributions-required-{PAGE}-{VIZ_ID}" for all. The change to move the identifying info to the front makes it a lot more legible

Reply:
yeah that's reasonable